From 2e5a780590c9af13fadca12d79da4d67ac5731b3 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Mon, 16 Oct 2023 11:39:29 +0200
Subject: [PATCH 10/15] configurations: sbp1: Prevent failsafe mode

FRU devices might be missing or powered off and onboard sensors might
not report sensor data. Mark all PID with the "InputUnavailableAsFailed"
set to false to prevent FailSafeMode in phosphor-pid-control when the
sensors doesn't return data.

Since there are so many sensors a single failure is tolerable.
Tested:

Change-Id: I9de1a4bdd61ac1bd8ddab1861d723315b8395b38
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_chassis.json | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/configurations/sbp1_chassis.json b/configurations/sbp1_chassis.json
index 142dc3a..9a9e3df 100644
--- a/configurations/sbp1_chassis.json
+++ b/configurations/sbp1_chassis.json
@@ -1584,6 +1584,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1612,6 +1613,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1656,6 +1658,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1685,6 +1688,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1711,6 +1715,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1751,6 +1756,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1791,6 +1797,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1831,6 +1838,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1871,6 +1879,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1897,6 +1906,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1923,6 +1933,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -1979,6 +1990,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -2004,6 +2016,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
@@ -2030,6 +2043,7 @@
             "SlewNeg": 0,
             "SlewPos": 0,
             "Type": "Pid",
+            "InputUnavailableAsFailed": false,
             "Zones": [
                 "PSU"
             ]
-- 
2.41.0

